.github/workflows/ci.yml

```diff
@@ -0,0 +1,66 @@
+name: "CI"
```

Comment:
_note to the reviewer:_ I'm not confident in the correctness of this script, since this is my first time writing it. ⚠️